doc/examples/features_detection/plot_brief.py

```diff
@@ -3,15 +3,12 @@
 BRIEF binary descriptor
 =======================
 
-This example demonstrates the BRIEF binary description algorithm.
-
-The descriptor consists of relatively few bits and can be computed using
-a set of intensity difference tests. The short binary descriptor results
-in low memory footprint and very efficient matching based on the Hamming
-distance metric.
-
-BRIEF does not provide rotation-invariance. Scale-invariance can be achieved by
-detecting and extracting features at different scales.
+This example demonstrates the BRIEF binary description algorithm. The descriptor
```

Comment:
If I remember correctly, our gallery shows the first sentense (or the first chapter(?)) in the popup.
Could you, please, check if this is rendered properly?
If not, I'd suggest to format the description so that only the first chapter is 1 sentence only.